src/PeopleSDKAdapter.js

```diff
@@ -184,4 +184,26 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
 
     return this.getPersonObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits a list of people that match the given query.
+   * An empty array is returned when there are no matches.
+   *
+   * @param {string} query Search query
+   * @returns {external:Observable.<Person[]>} Observable that emits person list based on search query
+   */
+  searchPeople(query) {
+    return defer(() => this.datasource.people.list({displayName: query}))
+      .pipe(
+        catchError((error) => {
+          logger.error('PEOPLE', undefined, 'searchPeople()', 'error fetching person list', error);
+          throw error;
```

Comment:
According to the description, this should return an empty array, not throw an error
```suggestion
          return [];
```
```suggestion
          throw error;
```